addons/hw_drivers/controllers/driver.py

```diff
@@ -79,7 +80,19 @@ def download_logs(self):
         """
```

Comment:
@xmo-odoo like most of the IoT routes :/ maybe it would be worth to discuss with the IoT team as a whole on the matter as it needs a heavy refactoring on the matter to be handled properly